proposals/0253-callable.md

```diff
@@ -415,35 +415,30 @@ add1(1, 2, 3)
 
 A type can both have `call` members and be declared with `@dynamicCallable`. When type-checking a call expression, the type checker will first try to resolve the call to a function or initializer call, then a `call` member call, and finally a dynamic call.
 
-### Direct reference to a `call` member
+### Using `call` members
 
-Like methods and initializers, a `call` member can be directly referenced, either through the base name and the contextual type, or through the full name.
+Currently, `call` members canot be directly referenced; create a thunk to call

```

Comment:
I don’t think “thunk” is ever mentioned in Swift’s language reference/guide, so we shouldn’t use it. Instead, we should use “closure”.

Reply:
Good point, done.